src/components/Tabs/index.tsx

```diff
@@ -70,6 +67,8 @@ const TabTitle = styled(GrommetTab)<{
 	}}
 `;
 
+export { InnerTabProps as TabProps, InnerTabsProps as TabsProps };
```

Comment:
Let's create `TabProps` that extends `GrommetTabProps` and adds children, and then the `InternalTabProps` extends `TabProps` and it adds `compact`. For the `TabsProps` we don't need separate internal typings cause there is nothing to add there. I know, it's confusing :)